### PR TITLE
Feature: replace console.log with purdy

### LINKED ISSUE #43: Setting as "default"?
Is there a way to have anything printed from node go through this?

Are there cons to that?

### PR DESCRIPTION
**Pull request by mightyiam**
Fixes #43

### CODE REVIEW

**Comment by danielb2** [comment]
This isn't going to work because purdy uses console.log, so it's going to call itself. that's why I mentioned process.stdout

also, be sure to follow the style I'm using which is based on hapi styleguides. I guess I should update the readme: https://github.com/hapijs/contrib/blob/master/Style.md

However, as I mentioned, Purdy doesn't even have the same syntax as console.log so you can't do this.

File: lib/index.js [hunk 1]
```diff
@@ -52,7 +57,7 @@ internals.purdy = function (object, options) {
 module.exports = function (object, options) {
 
     var Purdy = new internals.purdy(object, options);
-    return console.log(Purdy.stringify());
+    return consoleLog(Purdy.stringify());
```
Comment (mightyiam) [inline lib/index.js]: @danielb2 is this not the only place where Purdy calls `console.log`? Does it call `console.log` via dependencies, as well?

**Comment by mightyiam** [comment]
Perhaps `replace` could provide a converter between `console.log` signature and `Purdy` signature?

**Comment by danielb2** [comment]
I'll take a PR to replace console.log with process.stdout.write, that way you can override console.log as you please in your own code.

**Pull request closed without merging**